indico/modules/events/registration/blueprint.py

```diff
@@ -71,7 +72,12 @@
 
 _bp = IndicoBlueprint('event_registration', __name__, url_prefix='/event/<confId>', template_folder='templates',
                       virtual_template_folder='events/registration', event_feature='registration')
+_bp_participation = IndicoBlueprint('event_participation', __name__, url_prefix='/event/<confId>',
+                                    template_folder='templates', virtual_template_folder='events/registration')
 
+# Participants
+_bp_participation.add_url_rule('/manage/participants/',
+                               'manage_participants', RHManageParticipants, methods=('GET', 'POST'))
```

Comment:
Wouldn't everything besides the method fit in the first line? Also, i think `manage` would be enough for the endpoint name - we don't have anything else in that blueprint anyway...